packages/web-client/app/components/card-pay/issue-prepaid-card-workflow/face-value/index.hbs

```diff
@@ -13,7 +13,7 @@
         </span>
       </p>
       <CardPay::LabeledValue @vertical={{true}} @label="Funding From:">
-        <CardPay::IssuePrepaidCardWorkflow::FaceValue::BalanceView
+        <CardPay::BalanceViewBanner
```

Comment:
In our pairing, @Aierie and I decided to rename this `Layer2BalanceView` (as contrasted with the `BalanceView` within the deposit workflow that’s about layer 1), what do you think of that?

Reply:
I was thinking of using this component for that purpose too, but if it ends up being too complex, we can have two different ones that are named more specifically.